references/video_classification/train.py

```diff
@@ -119,11 +119,13 @@ def main(args):
     st = time.time()
     cache_path = _get_cache_path(traindir)
     transform_train = torchvision.transforms.Compose([
-        T.ToFloatTensorInZeroOne(),
+        bhwc_to_bchw,
```

Comment:
In general I would prefer to have a class to be passed around so that we have a consistent example for users on how to do things, and how we are mixing a function and a class within `Compose`. This works, but I wouldn't say this is something I would encourage users to do (and is also one of the reasons why we have `Lambda` transform.

I would just make this function into a class instantiate it here.